softgroup/basis/common-router/common-router-api/src/main/java/com/softgroup/common/router/api/factory/HandlerFactory.java

```diff
@@ -0,0 +1,18 @@
+package com.softgroup.common.router.api.factory;
+
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.router.api.Handler;
+
+/**
+ * Created by nikmlk on 26.02.17.
+ */
+
+public class HandlerFactory<T extends Handler> extends AbstractFactory<T>  implements Factory<T> {
```

Comment:
@melnyknikolay Name this for example CommandHandlerFactory. HandlerFactory is to general.